Delete template code, create basis for personal portfolio website with links, projects, and blog

zpChris opened the pull request:
Update README.md file.

zpChris commented:
I have created the foundation for my portfolio website, adding a personal image, a "background" section, and a favicon.

zpChris commented:
I have now added three projects with descriptions (Lambda AI, OFTW, and TeamsCode), and I added a **blog** page as well.

bheff-work commented on README.md:
```diff
@@ -1,5 +1,7 @@
 
+This repo contains Chris Elliott's portfolio and SPS projects.
+
 This repo contains the projects you'll work on for the first half of STEP.
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
```
No need to keep this extra text in here now that you've added your new description. You also don't need the header anymore.

bheff-work commented on portfolio/src/main/webapp/script.js:
```diff
@@ -12,17 +12,4 @@
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
```
The instructions for this project state that your webpage should be interactive: https://github.com/googleinterns/step/blob/master/walkthroughs/week-2-web-development/portfolio-walkthrough.md#javascript

So you don't want to delete all the javascript here, but modify it to do something different.

zpChris replied:
That makes sense, I'll change that now!

bheff-work commented on portfolio/src/main/webapp/index.html:
```diff
@@ -2,17 +2,53 @@
+      <img src="images/chris_profile.JPG" width="200px"/>
+      <h2>Background</h2>
+      <p>
+          I am a student in the University of Pennsylvania's Management and Technology Class of 2023, studying Computer Science and Economics.
```
At Google, we request that all lines be kept under 80 characters. One of the reasons for this is that if you look at the github ui, this line (and others) run off the edge of the box and require a slider. This makes it a lot more cumbersome to review.

We have handy linters that let us know when our code is too long, so we won't be super pedantic about this, but I'd love if you went back through and shortened a lot of these lines.

zpChris replied:
Ok, I'll look into linters that can help me keep my lines under 80 characters.

bheff-work replied:
No need to worry about it too much. The official policy for HTML is that breaking long lines is optional: go/htmlcssstyle#HTML_Line-Wrapping.

We're not expecting you to use linters or anything, so it's more of a best effort.

zpChris replied:
Ok, that makes sense. Also, just to let you know, I can't view the link `go/htmlcssstyle` (the style guide resources I've tried to access have unfortunately been blocked). (I asked my team, and Eshika cannot view this either.)

bheff-work replied:
oh bummer, that's lame. Okay, I'll quote the relevant text to you here then:

"Break long lines (optional).

While there is no column limit recommendation for HTML, you may consider wrapping long lines if it significantly improves readability.

When line-wrapping, each continuation line should be indented at least 4 additional spaces from the original line."

Thanks for letting me know that link doesn't work for you!

zpChris commented:
Ok, I removed the extra text in the `README.md` file, and added back in the JavaScript in `script.js` to add an interactive "fun fact" button. Next, I will add in a linter and modify the lines in `index.html` to keep them below 80 characters.

What is the process for "Resolving conversations" in GitHub? Should I wait for your feedback first, and then "Resolve" if everything is OK?

tristan-jh commented on portfolio/src/main/webapp/blog.html:
```diff
@@ -0,0 +1,67 @@
+        </p>
+        <div class="gallery-container">
+            <div class="gallery-image-container">
+                <img class="gallery-image" src="images/running.jpg"/>
```
It would be nice to add an alt attribute to these images. Alt attributes are usually picked up by screenreaders used by blind / vision impaired users.

More info in the google html/css style guide here: https://engdoc.corp.google.com/eng/doc/devguide/htmlcss/styleguide.md?cl=head#Multimedia_Fallback

zpChris replied:
Ok, sounds good!

Also, I noted this to Bootsie, but I'm not able to view style guides like the one you linked above, or ones at `go/htmlcssstyle`. I asked my teammates about this, and Eshika cannot view these either; they are unfortunately restricted access.

tristan-jh replied:
hmm, ok I'll follow up to see if we can get access for interns. In the mean time here's the public version which is mostly identical: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

bheff-work commented:
In response to resolving conversations, once you've addressed the concerns from a comment, then you can resolve the conversation. Once you're happy with the changes and are ready for me to look at it again, you should be able to re-request a review from me by clicking the circular errors by my name under 'reviewers'
